CoroutineTemplate/app/src/staging/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,22 @@
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <certificates src="user" />
+            <certificates src="system" />
+        </trust-anchors>
+    </debug-overrides>
+
+    <base-config cleartextTrafficPermitted="true">
+        <trust-anchors>
+            <certificates src="system" />
+        </trust-anchors>
+    </base-config>
+
+    <domain-config>
+        <domain includeSubdomains="true">www.jsonplaceholder.typicode.com</domain>
```

Comment:
🤔  So when we update the base URL then we have to update the domain as well. For now, I think we could add the comment to notice that replacing `includeSubdomains` to match the base URL. What do you think?